msi/rust.wxs

```diff
@@ -192,9 +201,9 @@
                         <Shortcut Id="RustShell"
                                   Name="$(var.ProductName) Shell"
                                   Description="Opens Command Prompt with Rust tools directory added to the PATH"
-                                  Target="[SystemFolder]CMD.exe"
+                                  Target="[SHORTCUTTARGET]"
```

Comment:
Was there a bug with `SystemFolder` not being set correctly?   Why is `%windir%\System32` better?
If you want to be pedantic about this, just use `%COMSPEC%` instead.